ttnn/cpp/ttnn/operations/eltwise/unary_backward/unary_backward_pybind.hpp

```diff
@@ -66,6 +66,9 @@ Keyword args:
                 }else if(operation.base_name()=="eq_bw"){
                     using BinaryBackwardOp = ttnn::operations::binary_backward::ExecuteBinaryBackward<binary_backward::BinaryBackwardOpType::EQ_BW>;
                     return BinaryBackwardOp::execute_on_worker_thread(grad_tensor, input_tensor_a, output_memory_config, input_tensor_b);
+                }else if(operation.base_name()=="sub_bw"){
```

Comment:
What is going on in this binding from line 47-70?

Reply:
why do we compare strings?
why do we call execute_on_worker_thread directly instead of calling ttnn:<op>?

Reply:
We did this to overload binary ops with unary ops @ayerofieiev-tt

Reply:
We have used this to achieve overloading by comparing the incoming op type and calling the suitable function in binary, we will try if is there any other better way to do this